src/index.html

```diff
@@ -7,11 +7,138 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>HTML Forms</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="block_input">
+        <legend>Personal information</legend>
+        <label class="line__input">
+          Surname:
+          <input
+          type="text"
+          autocomplete="off"
+          name="name">
```

Comment:
Pay attention on indents and closed bracket. Fix all places all over the page
```suggestion
          <input
            type="text"
            autocomplete="off"
            name="name"
          >
```